Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2661,6 +2665,54 @@ bool CSSPropertyParser::consumeScrollTimelineShorthand(bool important)
     return true;
 }
 
+bool CSSPropertyParser::consumeViewTimelineShorthand(bool important)
+{
+    CSSValueListBuilder namesList;
+    CSSValueListBuilder axesList;
+    CSSValueListBuilder insetsList;
+
+    auto defaultAxis = []() { return CSSPrimitiveValue::create(CSSValueBlock); };
+    auto defaultInsets = []() { return CSSPrimitiveValue::create(CSSValueAuto); };
+
+    do {
+        // A valid view-timeline-name is required.
+        if (auto name = CSSPropertyParsing::consumeSingleScrollTimelineName(m_range))
+            namesList.append(name.releaseNonNull());
+        else
+            return false;
+
+        // Both a view-timeline-axis and a view-timeline-inset are optional.
```

Comment:
This comment is a bit misleading, since one of two is required (according to the spec and what you implemented)

Reply:
From my reading of the spec for [view-timeline](https://drafts.csswg.org/scroll-animations-1/#propdef-view-timeline) the axis and insets are optional: `[ <'view-timeline-name'> [ <'view-timeline-axis'> || <'view-timeline-inset'> ]? ]#`. That certainly is what I implemented as well. What makes you think otherwise?

Reply:
Oh nevermind, I misread your code and the spec too. I missed the `?` from the spec.